### PR TITLE
Reporting: Subscription

### PR DESCRIPTION
**Pull request by klodav**

### CODE REVIEW

File: documentation/CHANGELOG-TMP.md [hunk 1]
```diff
@@ -7,6 +7,14 @@
 
 **Ticket    Date    Branch  Version(current) Author  Feature/Bug     Description/Keywords**
 
+3219    15.09.2021  rc2.0   2.0.4       David   Feature     Detailseiten für Zeitleisten (Lizenz-Reporting)
```
Comment (croqueGrec09) [inline documentation/CHANGELOG-TMP.md:10]: Time advanced fast ...

**Review by croqueGrec09 — COMMENTED** [review]
one remark, dates in changelog seem wrong

File: documentation/CHANGELOG-TMP.md (hunk 1 above)
Comment (klodav) [reply documentation/CHANGELOG-TMP.md]: :+1:

**Review by croqueGrec09 — APPROVED** [review]
approved

**Pull request merged**